etc/csp_whitelist.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<csp_whitelist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Csp:etc/csp_whitelist.xsd">
+    <policies>
+        <policy id="script-src">
+            <values>
+                <value id="cardknox-ifields-script" type="host">https://cdn.cardknox.com/ifields/2.6.2006.0102/ifields.min.js</value>
```

Comment:
It would be better if we have a way of using a wildcard for the end of the URL to avoid updating the file for every new version.

Reply:
The latest version may not be backwards compatible